Makefile

```diff
@@ -0,0 +1,15 @@
+NODE_BIN=./node_modules/.bin
+
+requirements:
+	npm install
+
+develop: requirements
+
+test: requirements
+	$(NODE_BIN)/karma start --single-run
+
+test-dev: requirements
+	$(NODE_BIN)/karma start
+
+quality: requirements
+	$(NODE_BIN)/jshint lib/ test/
```

Comment:
I asked this same question in the HipChat dev room the other day:

https://edx.hipchat.com/embedded/history/room/100299/2015/10/30?p=1&q=bower%20make&t=rid-100299#e28c3ab4-1a92-4911-8300-77092010ffc1

The consensus seems to be that ``make`` is lowest common denominator which works regardless of what your repo does -- ``paver`` is too Python specific, and ``gulp`` is too front end-specific. For a purely front end repo, I think it makes sense to use ``gulp`` exclusively without wrapping it with ``make``, but I can see the argument for using ``make`` just for consistency across all of our repos.